prov/verbs/src/fi_verbs.c

```diff
@@ -699,17 +736,20 @@ static int fi_ibv_read_params(void)
 	if (fi_ibv_get_param_int("dgram_device_port_number", "Port number of device to be "
 				 "only used for generating fi_info. The parameter is "
 				 "ignored if device name wasn't specified.",
-				 (size_t *)&fi_ibv_gl_data.dgram.device.port_number) ||
+				 &fi_ibv_gl_data.dgram.device.port_number) ||
 	    (fi_ibv_gl_data.dgram.device.port_number < 0 ||
-	     fi_ibv_gl_data.dgram.device.port_number > 32)) {
+	     fi_ibv_gl_data.dgram.device.port_number > 31)) {
 		VERBS_INFO(FI_LOG_CORE,
 			   "Invalid value of dgram_device_port_number\n");
 		return -FI_EINVAL;
 	}
 	if (fi_ibv_get_param_str("dgram_device_name", "The name of device to be "
 				 "only used for generating fi_info.",
-				 &fi_ibv_gl_data.dgram.device.name))
+				 &fi_ibv_gl_data.dgram.device.name)) {
+		VERBS_INFO(FI_LOG_CORE,
+			   "Invalid value of dgram_device_name\n");
```

Comment:
It would be better if these were warning logs instead of info.

Reply:
Ok, will do that